### ISSUE TITLE
Login to registry fails with "conn refused" on ubuntu 16.04, clean install 

### ISSUE THREAD

**Issue by ajvillarroelp** [minishift/minishift#2947]
### General information

  * Minishift version: minishift-1.25.0-linux
  * OS: Linux / macOS / Windows
Ubuntu 16.04  (elementary OS Loki)
  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox
KVM

This is a clean installation of minishift. I run minishift with root user.
Symptom is  that I  cannot build apps (pull or push error).
This command fails:

### Steps to reproduce

  1. minishift start
  2. 
root@av-Inspiron-5759:/opt/minishift-1.25.0-linux-amd64# oc whoami -t
bGGIdhOR_uwbuXcphGTkN9S-OVVjuWG59pKYm63UECw
root@av-Inspiron-5759:/opt/minishift-1.25.0-linux-amd64# minishift ssh
Last login: Fri Oct 26 08:39:40 2018 from 192.168.42.1
[docker@minishift ~]$ docker login -u developer -p bGGIdhOR_uwbuXcphGTkN9S-OVVjuWG59pKYm63UECw 172.30.1.1:5000
Error response from daemon: Get http://172.30.1.1:5000/v1/users/: dial tcp 172.30.1.1:5000: getsockopt: connection refused

### Logs
Start messages, docker ps command and above command are here:

https://gist.github.com/ajvillarroelp/d3e02404cf891e70115a3bc1190c9593

**Comment by ajvillarroelp** [comment]
Hi, 
Thinking about it is not a clean install. Before, I had to to reset minishift because it only works for root user. So I did:

minishift delete
virsh undefine <uuid>

Regards,

Antonio V.

**Comment by praveenkumar** [comment]
@ajvillarroelp does your issue is now resolved or you still facing it?

**Comment by ajvillarroelp** [comment]
Not solved.

**Comment by praveenkumar** [comment]
@ajvillarroelp I went through the logs https://gist.github.com/ajvillarroelp/d3e02404cf891e70115a3bc1190c9593 and looks like the registry pod was not even up yet. I think you need to wait for some time till pod is up and running something like below.

```console
$ docker ps | grep registry
9a164a711afb        a212cdb63bfb                                                                                                        "/bin/sh -c '/usr/..."   2 hours ago         Up 2 hours                              k8s_registry_docker-registry-2-l7xjc_default_b2e28c6a-d91a-11e8-918d-9ed3daed736a_0
ffe66e789d8d        openshift/origin-pod:v3.10.0                                                                                        "/usr/bin/pod"           2 hours ago         Up 2 hours                              k8s_POD_docker-registry-2-l7xjc_default_b2e28c6a-d91a-11e8-918d-9ed3daed736a_0
[docker@minishift bin]$ 

```

**Comment by ajvillarroelp** [comment]
Hi,

After doing a minishift delete --force and rm -rf ~/.minishift, and minishift start, 

docker ps -a
``
a5cf263d8e6a        docker.io/openshift/origin-deployer@sha256:704b5fea748bf7357649f55d5969cad631af20c3b8157d8d4add4c6f4610f0d0        "/usr/bin/openshif..."   2 minutes ago        **Exited (1) 2 minutes ago**                            k8s_deployment_docker-registry-1-deploy_default_38e36681-db67-11e8-b9bc-525400f3d6d9_0
``

The registry pod exited with error. This is the log for this registry:

``
error: couldn't get deployment docker-registry-1: replicationcontrollers "docker-registry-1" is forbidden: User "system:serviceaccount:default:deployer" cannot get replicationcontrollers in the namespace "default": User "system:serviceaccount:default:deployer" cannot get replicationcontrollers in project "default"
``

Any ideas?

Thank you.

Antonio V.

**Comment by ajvillarroelp** [comment]
Hi,

I just saw that a new minishift version is available for download (1.26). so I downloaded, and tried it again. I am happy to say that the problem is solved.

Kind regards,

Antonio V.
